apps/bot_manager/lib/application.ex

```diff
@@ -7,21 +7,23 @@ defmodule BotManager.Application do
 
   @impl true
   def start(_type, _args) do
-    http_server =
-      Plug.Cowboy.child_spec(
-        scheme: :http,
-        plug: BotManager.Endpoint,
-        options: Application.get_env(:bot_manager, :end_point)[:port]
-      )
+    options = [port: get_bot_manager_port()]
```

Comment:
Don't get the reasoning for this instead of configs, can you explain a bit?

Reply:
basically i didn't understood the benefit of having the configs there instead of just creating them here, i moved this to the runtime.ex file and it worked 😄 i just moved the whole endpoint config there so we centralize that